kitsune/questions/config.py

```diff
@@ -330,10 +332,10 @@
                             },
                         ),
                         (
-                            "manage-add-ons-enterprise",
+                            "install-and-manage-add-ons",
                             {
-                                "name": _lazy("Manage add-ons"),
-                                "topic": "manage-add-ons-enterprise",
+                                "name": _lazy("Install and manage add-ons"),
+                                "topic": "install-and-manage-add-ons",
                                 "tags": ["customization"],
```

Comment:
Should the `tags` changes as well?
```suggestion
                                "tags": ["install-and-manage-add-ons"],
```